content/docs/contributing/release-process.md

```diff
@@ -308,7 +308,18 @@ page if a step is missing or if it is outdated.
         only to avoid rate-limits imposed on anonymous API users.
         </p></div>
 
-    3. Sanity check the notes:
+    3. Replace the GitHub issue numbers and GitHub handles (e.g., `#1234` or
+       `@maelvls`) with actual links:
+
+       ```bash
+       sed release-notes.md \
```

Comment:
Nit: the filename will actually be `github-release-description.md` as per later changes in this PR